### DISCUSSION TITLE
Any video how to do Visual Studio e.g. VS2022 setup?

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [MikeSchulze/gdUnit4Mono#28] [Q&A] [answered]
Is VS2022 possible?

**Comment by MikeSchulze** [comment]
Can you please describe in more detail what you mean?

**Reply by GeorgeS2019** [reply]
It is not clear what is the experience using Visual Studio Code. 

It is not clear if the same experience could also happen to VS2022. 

Since there is no Video related to VS Code or VS2022, I wonder if the experience in VS2022 is similar to e.g. NUnit

**Reply by GeorgeS2019** [reply]
If is unclear if the scene running experience is also working through VS Code. Most would assume, it is limited to within gdUnit4 UI within GODOT.

**Reply by GeorgeS2019** [reply]
Will this VSCode experience also work with VS2022?

https://mikeschulze.github.io/gdUnit4/faq/C%23/#running-c-tests-by-visual-studio-test-adapter

![image](https://github.com/MikeSchulze/gdUnit4Mono/assets/49812372/91c35308-5ebb-4bcf-805c-3c7a9aee06a9)

**Comment by MikeSchulze** [comment, marked as answer]
Hi @GeorgeS2019 in the documentation there are still outdated information about the c# test adapter (GdUnit3 test adapter).
I'm actual working on a new [test adapter for GdUnit4](https://github.com/MikeSchulze/gdUnit4Mono/tree/master/testadapter) but is not released yet.


The new test adapter implements [vs-test API ](https://github.com/microsoft/vstest/blob/main/docs/RFCs/0004-Adapter-Extensibility.md), which are supported by Microsoft-VS, Microsoft-VSC and JetBrains-Rider.

**Comment by GeorgeS2019** [comment]
using the latest gdUnit4API **TestAdaptor** on VS2022

![Z0nhScGr3J](https://github.com/MikeSchulze/gdUnit4Mono/assets/49812372/7d2ed607-0750-4b6e-80f7-e5913aec040e)

**Comment by MikeSchulze** [comment]
All README are updated.
